src/Controller/CreateProjectController.php

```diff
@@ -135,19 +133,37 @@ public function createProject(Request $request): Response
      */
     public function submitted(Request $request): Response
     {
+        $formData = $_SESSION['form_data'];
```

Comment:
No fan of using the super globals anymore... see https://symfony.com/doc/current/session.html

> Symfony sessions are designed to replace the usage of the $_SESSION super global and native PHP functions

Reply:
Will fix